lib/_http_client.js

```diff
@@ -387,7 +389,7 @@ function _destroy(req, socket, err) {
     if (err) {
       req.emit('error', err);
     }
-    req._closed = true;
+    req[kClosed] = true;
```

Comment:
This is not ok, see discussion on https://github.com/nodejs/node/pull/34313.

Reply:
This seems strange, I can't remember changing this part of the file however I will revert the changes for now and create another PR.

Reply:
`git pull --rebase`